### PR TITLE
Add observables for smelli 2.0

### PR DESCRIPTION
**Pull request by DavidMStraub**
Related: #20 

This adds:
- Higgs observables
- ee->WW (LEP)
- neutron beta decay (lifetime & correlations)
- superallowed beta decays (Ft values)
- semi-leptonic kaon decays

The SM covariances are also updated. The flavio requirement is updated to 2.0.0. The smelli version is also set to 2.0.0.

### CODE REVIEW

File: smelli/data/yaml/observables_n.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+- name: tau_n
+  me_E: 0.655
+- name: Atilde_n
+  me_E: 0.559
+- name: Atilde_n
```
Comment (peterstangl) [inline smelli/data/yaml/observables_n.yaml:5]: @DavidMStraub there are two entries for `Atilde_n`. Which one is the correct one?

Comment (DavidMStraub) [reply smelli/data/yaml/observables_n.yaml:5]: Are they both in Flavio? Could be just two experiments with different kinematics.

Comment (peterstangl) [reply smelli/data/yaml/observables_n.yaml:5]: Ah, yes that's the reason. UCNA has `me_E=0.586` and PERKO has `me_E=0.559`.

**Comment by peterstangl** [comment]
@DavidMStraub I think this looks good! I already merged some other PRs and updated the flavio version (but not the smelli version yet) to make the tests pass. Can you please rebase this branch on the master branch?
Due to changes in https://github.com/smelli/smelli/commit/fd12b8bb0f1ca999f44b02929d542cecca636ee6 and https://github.com/smelli/smelli/commit/947d9e00bddb4d9893834fdc2d688c09e629a92d, the file names of the cached sm covariances have changed (to allow for different files in case of different CKM schemes and also separate files for fixed CKM). The easiest way to get all files is maybe to just rerun the generation of sm covariances. I can take care of this as soon as this branch is rebased on master.

**Comment by DavidMStraub** [comment]
Rebased on master now. I cannot generate SM covariances today anymore, so if you can do it that would be great.

**Comment by peterstangl** [comment]
Great, thanks! I will take care of the SM covariances.

**Pull request merged**